packages/bin-etl/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import fs from 'fs'
+
+const fileName = './worlds.json'
+processFile(fileName)
+
+function processFile(n: string) {
+    fs.readFile(n, 'utf8', function (err,data) {
+        if (err) {
+          return console.log(err);
+        }
+        handleJson(JSON.parse(data))
+      });
+}
+
+function handleJson(data: any) {
+    const cluster = data["world"]["clusters"]["cluster"]
+    const selectedFields = new Array()
+    for (let m of cluster) {
```

Comment:
This for loop can be done with `Array.map` for a functional approach, instead of pushing to a separately created array.